etl/paths.py

```diff
@@ -5,6 +5,7 @@
 DAG_DIR = BASE_DIR / "dag"
 DAG_FILE = DAG_DIR / "main.yml"
 DAG_ARCHIVE_FILE = DAG_DIR / "archive" / "main.yml"
+DAG_TEMP_FILE = DAG_DIR / "temp.yml"
```

Comment:
I don't see this being used in other files. Maybe you forgot to undo this?

Reply:
`DAG_TEMP_FILE` is used by `apps/step_update/cli.py`.